lib/darksky.rb

```diff
@@ -0,0 +1,33 @@
+require 'httparty'
+
+class Darksky
+  def self.darksky_info(coordinates)
```

Comment:
I don't normally use class methods often, but it felt odd using instance methods when I couldn't really explain what "an instance of Darksky" would represent. Maybe it was just a naming problem, and if I had named the class "Weather" it would have made more sense to me. To me it seemed like these should be modules, but I haven't written modules independently of classes before, so I chose to not to burn too much time experimenting with that for the first time here.
If I were saving data to a database I'd use instances, or if I were expecting to have to distinguish between multiple function calls in a session.